docs/API.md

```diff
@@ -303,6 +304,11 @@ If you are using CoffeeScript this should be `['js', 'json', 'coffee', 'litcoffe
 
 An array of regexp pattern strings that are matched against all module paths before those paths are to be considered 'visible' to the module loader. If a given module's path matches any of the patterns, it will not be `require()`-able in the test environment.
 
+### `config.preprocessCachingDisabled` [boolean]
+(default: `false`)
+
+While on disk caching is enabled by default, this options allows to explicitly disabled it. Turning caching off is helpful while developing preprocessor scripts, to avoid getting cached outputs.
```

Comment:
"Disables caching for preprocessed files. The preprocess cache is a performance optimization and should not be disabled unless there are unresolvable issues with reading/writing files from the cache directory."

(I'll be adding --nocache soon, so I don't think the second sentence is useful :) Feel free to send a pull request if you want to add this option yourself!).